pallets/asset-registry/src/lib.rs

```diff
@@ -556,6 +556,14 @@ impl<T: Config> Pallet<T> {
 			},
 		)
 	}
+
+	#[allow(clippy::ptr_arg)]
```

Comment:
Do we need this here? Doesn't seem like any pointer arg is passed to this function.

Reply:
fixed